### PR TITLE
[N/A] Add info to README

### PR DESCRIPTION
**Pull request by pedrofelipe**
- [x] Add info to README
- [x] Add screenshot
- [x] Change `.gitignore` to ignore new `public` folder

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,44 @@
...
+```bash
+gulp
+```
+
+### Tests
+
+Run to execute the tests in the browser
+
+```bash
+gulp serve:test
```
Comment (viniciusalmeida) [inline README.md]: I guess we can use just `gulp test`, right?

**Review by viniciusalmeida — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (pedrofelipe) [reply README.md]: @viniciusalmeida since they're both working (cli and browser), I'll add them all to readme, ok? 👍

Comment (pedrofelipe) [reply README.md]: @viniciusalmeida done!

**Comment by gabrielrinaldi** [comment]
Can you review/merge my pull request, then rebase yours? That will allow the tests to pass

**Comment by pedrofelipe** [comment]
done

**Pull request merged**